nipype/interfaces/ants/tests/test_registration.py

```diff
@@ -0,0 +1,21 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+from __future__ import unicode_literals
+from nipype.interfaces.ants import registration 
+import os
+import pytest
+
+def test_ants_mand():
```

Comment:
does the interface actually create files? if so (and I guess it would), we should include `tmpdir.chdir()`

Reply:
done